lighthouse-core/driver/gatherers/offline.js

```diff
@@ -38,23 +38,25 @@ const requestPage = function() {
 
 class Offline extends Gather {
 
-  static goOffline(driver) {
-    return driver.sendCommand('Network.emulateNetworkConditions', {
-      offline: true,
+  static networkConfig(bool) {
+    return {
+      offline: bool,
```

Comment:
also this is a boolean trap we should use an object here and destructure it for things to be much more readable on the consumer and caller side